modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/utilities/forms/DDMFormHandler.js

```diff
@@ -68,6 +70,37 @@ class DDMFormHandler {
 			}
 		});
 	}
+
+	checkCPInstanceOptions() {
+		var emptyRequiredField = false;
+		const addToCartButton = document.querySelector('.btn-add-to-cart');
+		const ddmFormValues = JSON.stringify(this.fields);
+
+		for (const ddmFormValue of JSON.parse(ddmFormValues)) {
+			if (ddmFormValue.required === true) {
+				for (const val of ddmFormValue.value) {
+					if (val === '' || val === null) {
```

Comment:
We want to whether the variable si truthy or falsy so `if (val) {` is shorter and works the same